CreeDictionary/DatabaseManager/__main__.py

```diff
@@ -48,18 +49,32 @@ def add_multi_processing_argument(ap: ArgumentParser):
 add_multi_processing_argument(import_parser)
 
 
+def import_and_migrate(xml_dir_name: Union[Path, str], process_count: int):
+    """
+    Unlink existing .sqlite3 file, apply migrations from API/0001 to API/0005
+    Import from specified engcrk.xml and crkeng.xml. Migrate the rest from all apps
+    """
+    try:
+        Path(settings.DATABASES["default"]["NAME"]).unlink()
+    except FileNotFoundError:
+        pass
+    call_command("migrate", "API", "0005")
```

Comment:
Why specifically are you hardcoding migrating 005?

Reply:
0005 is the migration to date. This makes sense since we switched to the workflow mid way. Say if we started with this workflow, we'd still need to migrate 0001_initial.py to create a well formed database at first, then import all the words, and then all the rest migrations.